use-location.js

```diff
@@ -1,5 +1,10 @@
 import { useEffect, useRef, useState, useCallback } from "./react-deps.js";
 
+const eventPopState = "popstate"
+const eventPushState = "pushstate"
+const eventReplaceState = "replaceState"
+export const eventsHistory = [eventPopState, eventPushState, eventReplaceState]
```

Comment:
Was thinking about a shorter name for this. Maybe just `events`? Exported constant names can't be minified, so the length matters 😃

Reply:
Could you please clarify why you think mangling won't work during minification?
In my experience and just checked some bundles minified with [terser](https://github.com/terser/terser) and don't see an issue w/ mangling.

Reply:
Didn't think about that, makes sense!